libs/dal/src/repositories/message/message.repository.ts

```diff
@@ -624,6 +624,7 @@ export class MessageRepository extends BaseRepository<MessageDBModel, MessageEnt
     } else if (isUpdatingSnoozed) {
       updatePayload = {
         snoozedUntil,
+        read: !snoozedUntil ? false : undefined,
```

Comment:
if notification gets manually unsnoozed then it doesn't need to be unread

Reply:
we also need to update the `readAt` flag; but I'm not sure when I do manually unsnooze the `read` flag IMO shouldn't change, but when the job unsnoozes it should be `read: false`

Reply:
Yes, I pushed new changes - it should behave exactly as you said.